### PR TITLE
Create ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-55911](https://cumulocity.atlassian.net/browse/MTM-55911)
Original PR: [5630](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5630)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+date: ""
+title: ui refresh button on microservice status exclude smartrules (#5522) [GRAFT][release/cd] (#5630)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Updated User Interface Refresh Button to Exclude SmartRules Microservice Status
```

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-55911
+version: 1019.6.9
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:17]: The Claude AI suggested to use the following body:
```suggestion
The user interface refresh button was updated to exclude the SmartRules microservice status. This improves page load performance by not making an unnecessary call to retrieve the SmartRules status when refreshing the page. 

This change was contributed by Pawel Michalowski.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+---
+The user interface refresh button on the microservice detail view was updated to exclude the SmartRules microservice status.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:17]:
```suggestion
The **Refresh** button on the microservice details page does no longer include the smart rules microservice status.
```

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Updated user interface refresh button to exclude SmartRules microservice status
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:3]:
```suggestion
title: Refresh button no longer includes smart rules microservice status
```

Comment (BeateRixen) [reply content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:3]: Please change the file name accordingly

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md (hunk 3 above)
Comment (pmich-sag) [inline content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:17]:
```suggestion
The **Refresh** button on the microservice status details page includes the smart rules list.
```

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md (hunk 4 above)
Comment (pmich-sag) [inline content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:3]:
```suggestion
title: Refresh button includes smart rules list in microservice status page
```

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-55911
+version: 1019.6.9
+---
+The **Refresh** button on the microservice status details page includes the smart rules list.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:17]:
```suggestion
The **Refresh** button on the **Status** tab of the microservice details now includes the smart rules list.
```

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Refresh button includes smart rules list in microservice status page
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md:3]:
```suggestion
title: Refresh button on microservice status tab includes smart rules list
```

**Review by BeateRixen — APPROVED** [review]

**Review by pmich-sag — APPROVED** [review]

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md (hunk 4 above)
Comment (pmich-sag) [reply content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md]: Done.

File: content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md (hunk 3 above)
Comment (pmich-sag) [reply content/change-logs/ui-c8y-1019-6-9-ui-refresh-button-on-microservice-status-exclude-smartrules-5522-graft-release-cd-5630-co-authored-by-pawel-michalowski-pmic-softwareag-com.md]: Done.

**Pull request merged**